WordPress/wordpress-playground
# Include CORS proxy with website builds

## Motivation for the change, related issues

This PR updates the Playground website build to include the CORS proxy.

The proxy is not quite ready for broad use yet. The following issues need to be addressed:
- [x] Add rate-limiting to discourage abuse (#1879)
- [ ] Troubleshoot CORS proxy errors in Safari (mentioned [here](https://github.com/WordPress/wordpress-playground/pull/1865#issuecomment-2401184209))

But, @adamziel, if desired and if you are comfortable with the risk, you could merge this and deploy the CORS proxy now to enable initial testing.

## Implementation details

This updates nx project descriptions to include php-cors-proxy scripts with the Playground website builds.

## Testing Instructions (or ideally a Blueprint)

- Run `npx nx build playground-website`
- Run `ls dist/packages/playground/wasm-wordpress-net/ | grep cors-proxy` and confirm `cors-proxy.php` and `cors-proxy-functions.php` are present.
